### DISCUSSION TITLE
Error "Uncaught PHP Exception" in Log

### DISCUSSION THREAD

**Discussion by WhiteTiger-IT** [kimai/kimai#2328] [Questions & Answers]
I am using Kimai without difficulty.
However in the log I find this error repeated.

`[2021-02-08 12:00:26] request.CRITICAL: Uncaught PHP Exception Twig\Error\RuntimeError: "An exception has been thrown during the rendering of a template ("Failed to create "/var/www/html/time.MYDOMAIN/public_html/kimai2/var/cache/prod/translations": mkdir(): Permission denied")." at /var/www/html/time.MYDOMAIN/public_html/kimai2/templates/base.html.twig line 68 {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): An exception has been thrown during the rendering of a template (\"Failed to create \"/var/www/html/time.MYDOMAIN/public_html/kimai2/var/cache/prod/translations\": mkdir(): Permission denied\"). at /var/www/html/time.MYDOMAINe/public_html/kimai2/templates/base.html.twig:68, Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Failed to create \"/var/www/html/time.MYDOMAIN/public_html/kimai2/var/cache/prod/translations\": mkdir(): Permission denied at /var/www/html/time.MYDOMAIN/public_html/kimai2/vendor/symfony/filesystem/Filesystem.php:105)"} []
`

**Comment by kevinpapst** [comment]
https://github.com/kevinpapst/kimai2/discussions/2326

**Reply by WhiteTiger-IT** [reply]
As I said, Kimai is working and the rights had already been applied at least 4 hours before; after which I have not added any Plugins anymore.
I accidentally noticed the error in the log.

**Reply by kevinpapst** [reply]
Our last discussion in #2326 show the error at `[2021-02-08 11:40:04]` and at 12:05 you posted that it is now working. The message above is from `[2021-02-08 12:00:26]`. So I assume this can safely be ignored.

**Reply by WhiteTiger-IT** [reply]
Definitely I have to start doing less things together.
I find that I am no longer multitasking.
Sorry, I hadn't seen that the log was on an old and already fixed issue.
